docs/codeql/codeql-for-visual-studio-code/running-codeql-queries-at-scale-with-mrva.rst

```diff
@@ -153,6 +153,32 @@ For example, if you want to continue analyzing a set of repositories that had re
 
 You can then insert the ``new-repo-list`` of repositories into your list of custom repository lists for easy access in the Variant Analysis Repositories panel.
 
+Using code search to add repositories to a custom list 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can use code search directly in the CodeQL extension to add a subset of repositories from GitHub.com to a custom list. Note that this feature uses the legacy code search via the code search API. For more information on the syntax to use, see "`Searching code (legacy) <https://docs.github.com/en/search-github/searching-on-github/searching-code>`__."
+
+For example, to add all repositories in the ``rails`` organization on GitHub, you can search ``org:rails``. 
```

Comment:
@norascheuch @jf205 hello! I was wondering if there is a certain example search that might be helpful to add here? I did a very basic one but not sure if there is a name of a certain function or search term that might be helpful to illustrate the benefits. Thanks!

Reply:
I think using this as an example is fine! I have been testing with some library names, but I don't know enough about the ecosystem to know which ones would be a better idea to share in the docs. (I used `SQLAlchemy` and `RuntimeExec` for example)